util/log/clog.go

```diff
@@ -45,66 +45,73 @@ import (
 // the flag.Value interface. The -stderrthreshold flag is of type Severity and
 // should be modified only through the flag.Value interface. The values match
 // the corresponding constants in C++.
-type severity int32 // sync/atomic int32
+type Severity int32 // sync/atomic int32
 
 // These constants identify the log levels in order of increasing Severity.
 // A message written to a high-Severity log file is also written to each
 // lower-Severity log file.
 const (
-	infoLog severity = iota
-	warningLog
-	errorLog
-	fatalLog
-	numSeverity = 4
+	InfoLog Severity = iota
+	WarningLog
+	ErrorLog
+	FatalLog
+	NumSeverity = 4
 )
 
 const severityChar = "IWEF"
 
 // SeverityName provides a mapping from Severity level to a string.
-var severityName = []string{
-	infoLog:    "INFO",
-	warningLog: "WARNING",
-	errorLog:   "ERROR",
-	fatalLog:   "FATAL",
+var SeverityName = []string{
```

Comment:
I think you're right, it doesn't need to be exported.  I unexported it.

And doesn't stringer add .String(), which is already used by Severity to display the number?